frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/advanced-tab/sysprep/sysprep-autounattend/SysprepAutounattendHelperPopup.tsx

```diff
@@ -11,10 +11,12 @@ const SysprepAutounattendHelperPopup: React.FC = () => {
         <Trans t={t} ns="kubevirt-plugin">
           <Text component={TextVariants.h6}>Autounattend.xml</Text>
           <Text component={TextVariants.p}>
-            The answer file can be provided in a ConfigMap or a Secret with the key autounattend.xml
+            Autounattend will be picked up automatically during windows installation. it can be used
+            with destructive actions such as disk formatting. Autounattend will only be used once
+            during installation.
           </Text>
           <ExternalLink
-            href="https://kubevirt.io/user-guide/virtual_machines/startup_scripts/#sysprep"
+            href="https://docs.microsoft.com/en-us/windows-hardware/customize/desktop/wsim/answer-files-overview"
```

Comment:
we should not link to microsoft docs from the UI, openshft docs are best and if we don't have any, kubevirt

Reply:
returned the link to kubevirt